talent/engineering-teams-descr.md

```diff
@@ -0,0 +1,42 @@
+This document describes the engineering teams at source{d} and how they interact with each other.
+
+### Machine Learning Team
+
+- The Machine Learning team is focused on providing intelligent insights into nearly all the world's Open Source code. This includes typical Big Data analysis as well as solving sophisticated Machine Learning problems. The former is conducted on Apache Spark clusters, the latter leverages extensive GPGPU acceleration on custom hardware using deep learning techniques. Often ML problems we face are unsupervised and require novel research in the domain of ML on source code (MLonCode).
+
+### Applications Team
+
+- The Applications team applies the whole stack and expertise to solve real word problems as code duplication, smart diffs, search for repositories, projects evolving reports. This is done by implementing CLI/Web applications on top of the MLonCode research as well as the other teams main tools, like BabelFish, Engine, GitBase etc. Simple and intuitive applications are written mainly in Go and Scala, based on developer tooling prototypes.
```

Comment:
Sorry for confusion, but as it's the first time I hear about it - wondering, what are the  `developer tooling prototypes`?

Reply:
@bzz I am looking at the Applications team (SF and Madrid) as a whole and one team as they actually are as far as I know. Taking into account their descriptions https://github.com/src-d/guide/blob/master/talent/job-descriptions/offers/applications-sf.md and https://github.com/src-d/guide/blob/master/talent/job-descriptions/offers/applications.md
it seems that the Applications team is building developer tools based on ML/LA work as well as on the prototypes built by SF. As far as I understand, at the end of the day, what Applications team is doing is building Developer Tools. Correct me if I am wrong and feel free to provide the actual description as you and your team know better.

Reply:
Thank you for explaining.

>  one team as they actually are as far as I know
Yes, that's something that has been discussed so far. Was confused by two entities on the diagram though.

> as well as on the prototypes built by SF

I think that might not be something we have decided up on, and thus I would suggest removing this, to avoid the confusion.

Just `Simple and intuitive applications are written mainly in Go and Scala.` should be enough.